nsot/admin.py

```diff
@@ -46,15 +46,15 @@ class AttributeAdmin(admin.ModelAdmin):
 admin.site.register(models.Attribute, AttributeAdmin)
 
 
-class ValueAdmin(admin.ModelAdmin):
+class ValueAdmin(GuardedModelAdmin):
     list_display = ('name', 'value', 'resource_name', 'resource_id')
     list_filter = ('name', 'value', 'resource_name')
 
 
 admin.site.register(models.Value, ValueAdmin)
 
 
-class ChangeAdmin(admin.ModelAdmin):
+class ChangeAdmin(GuardedModelAdmin):
```

Comment:
Since `Change` objects are read-only we don't need to modify this base to allow for manipulation of object-level permissions from the UI.

Reply:
Done.